test/functional/p2p_addr_relay.py

```diff
@@ -75,6 +79,19 @@ def setup_addr_msg(self, num):
         msg.addrs = addrs
         return msg
 
+    def setup_rand_addr_msg(self, num):
```

Comment:
I tried to reuse it, but things fail because the existing setup_addr_message are limited to 256 addresses, and when I tried to generalize it, some of the other tests started failing.